nixos/tests/rosenpass.nix

```
@@ -0,0 +1,164 @@
+import ./make-test-python.nix ({ pkgs, ... }:
+let
+  deviceName = "rp0";
+
+  server = {
+    ip = "fe80::1";
+    wg = {
+      public = "mQufmDFeQQuU/fIaB2hHgluhjjm1ypK4hJr1cW3WqAw=";
+      secret = "4N5Y1dldqrpsbaEiY8O0XBUGUFf8vkvtBtm8AoOX7Eo=";
+      listen = 10000;
+    };
+  };
+  client = {
+    ip = "fe80::2";
+    wg = {
+      public = "Mb3GOlT7oS+F3JntVKiaD7SpHxLxNdtEmWz/9FMnRFU=";
+      secret = "uC5dfGMv7Oxf5UDfdPkj6rZiRZT2dRWp5x8IQxrNcUE=";
+    };
+  };
+in
+{
+  name = "rosenpass";
+
+  nodes =
+    let
+      etcPath = config: name: "/etc/" + config.environment.etc.${name}.target;
+
+      shared = peer: { config
+                     , modulesPath
+                     , ...
+                     }: {
+        imports = [ "${modulesPath}/services/networking/rosenpass.nix" ];
```

```suggestion
      shared = peer: { config, modulesPath, ... }: {
        imports = [ "${modulesPath}/services/networking/rosenpass.nix" ];
```
I wouldn't allign here to keep the code shorter

---

I am pretty sure it looked like that when I defined it, and then `nixpkgs-fmt` changed it...